nat-lab/network.md

```diff
@@ -0,0 +1,217 @@
+```mermaid
+graph LR
+%% AUTO GENERATED DIAGRAM. To update run `./utils/generate_network_diagram.py docker-compose.yml network.md`
```

Comment:
Please add a github action step that runs this command and verifies that it is successful.

Reply:
Good idea! I'll look into it.